pkg/apis/servicecatalog/types.go

```diff
@@ -191,7 +191,7 @@ type ServiceClass struct {
 
 	// Plans is the list of ServicePlans for this ServiceClass.  All
 	// ServiceClasses have at least one ServicePlan.
-	Plans []ServicePlan
+	Plans []Plan
```

Comment:
@duglin 
I don't know what that is. Do you mean the `BrokerName`? Shouldn't that be some kind of `ObjectReferece`?

It sounds like you are disagreeing with the reference direction proposed.